script/tasks/diamondUpdateFacet.sh

```
@@ -116,27 +115,48 @@ diamondUpdateFacet() {
     # check if we are deploying to PROD
     if [[ "$ENVIRONMENT" == "production" && "$SEND_PROPOSALS_DIRECTLY_TO_DIAMOND" != "true" ]]; then
-      # PROD (normal mode): suggest diamondCut transaction to SAFE
+      # PROD: suggest diamondCut transaction to SAFE
+
       PRIVATE_KEY=$(getPrivateKey "$NETWORK" "$ENVIRONMENT")
-      echoDebug "Calculating facet cuts for $CONTRACT_NAME with script $SCRIPT_PATH..."
+      echoDebug "Calculating facet cuts for $SCRIPT..."
 
       if isZkEvmNetwork "$NETWORK"; then
-        RAW_RETURN_DATA=$(FOUNDRY_PROFILE=zksync NO_BROADCAST=true NETWORK=$NETWORK FILE_SUFFIX=$FILE_SUFFIX USE_DEF_DIAMOND=$USE_MUTABLE_DIAMOND PRIVATE_KEY=$PRIVATE_KEY ./foundry-zksync/forge script "$SCRIPT_PATH" -f "$NETWORK" -vvvv --json --skip-simulation --slow --zksync)
+        RAW_RETURN_DATA=$(FOUNDRY_PROFILE=zksync NO_BROADCAST=true NETWORK=$NETWORK FILE_SUFFIX=$FILE_SUFFIX USE_DEF_DIAMOND=$USE_MUTABLE_DIAMOND PRIVATE_KEY=$PRIVATE_KEY ./foundry-zksync/forge script "$SCRIPT_PATH" -f $NETWORK -vvvv --json --skip-simulation --slow --zksync)
```

Shellcheck (0.10.0)

[warning] This assignment is only seen by the forked process.

(SC2097)